libs/utils/android/benchmark.py

```diff
@@ -203,9 +211,15 @@ def __init__(self):
             raise
 
         self._log.info('=== Execution...')
-        self.wl.run(out_dir=self.out_dir,
-                    collect=self._getBmCollect(),
-                    **self.bm_params)
+        for iter in range(self.bm_iterations):
```

Comment:
Hummm.. your idea is certainly more elegant.
Couldn't we want however to be able to configure sleep/reboot from Jenkins as well though?
Maybe we can add both ways of configuring sleep/reboot?